### DISCUSSION TITLE
the ItemsControl elements leave when the window opens WindowState="FullScreen"

### DISCUSSION THREAD

**Discussion by waxmask** [AvaloniaUI/Avalonia#15986] [Q&A]
### Describe the bug

Windows 10. Avalonia 11.0
I am developing a windows application, I follow the mvvm pattern as shown in the todolist instructions. I would like to place the list of items in the ItemsControl, but if I set WindowState="FullScreen" in the axaml file, my items leave.

### To Reproduce

the code for my view:
```<UserControl xmlns="https://github.com/avaloniaui"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
             xmlns:vm="using:RobotCarWashGUI.ViewModels"
             x:DataType="vm:MainViewModel"
             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
             x:Class="RobotCarWashGUI.Views.MainView">

    <Grid RowDefinitions="auto * auto" Margin="10">

        <StackPanel Grid.Row="2">
            <Border HorizontalAlignment="Center"
                    Background="#fff"
                    Padding="15 15 15 0"
                    CornerRadius="15 15 0 0">
                <TextBlock FontWeight="Bold" FontSize="24">Перед мойной убедитесь</TextBlock>
            </Border>

            <Border HorizontalAlignment="Center"
                    Background="#fff"
                    Padding="15 "
                    CornerRadius="15 ">
                <ItemsControl ItemsSource="{Binding Descriptions}">
                    <ItemsControl.ItemTemplate>
                        <DataTemplate>
                            <TextBlock Margin="10"
                                       TextAlignment="Center"
                                       FontSize="24"
                                       TextWrapping="Wrap"
                                       Text="{Binding Text}">
                            </TextBlock>
                        </DataTemplate>
                    </ItemsControl.ItemTemplate>
                </ItemsControl>
            </Border>

            <Button Margin="0 20 0 0"
                    HorizontalAlignment="Stretch"
                    VerticalAlignment="Stretch"
                    Height="40"
                    Classes="AppButton">
                Далее
            </Button>
        </StackPanel>
    </Grid>
</UserControl>```

as well as the code of my window
```
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:vm="using:RobotCarWashGUI.ViewModels"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        x:Class="RobotCarWashGUI.Views.MainWindow"
        x:DataType="vm:MainWindowViewModel"
        Icon="/Assets/avalonia-logo.ico"
        Title="RobotCarWashGUI"
        WindowState="FullScreen"
        Content="{Binding CurrentView}">

    <Window.Background>
        <ImageBrush Source="{Binding ApplicationBackgroundImage}" Stretch="UniformToFill"></ImageBrush>
    </Window.Background>

</Window>
```

### Expected behavior

_No response_

### Avalonia version

11.0.10

### OS

Windows

### Additional context

_No response_

**Comment by waxmask** [comment]
![image](https://github.com/AvaloniaUI/Avalonia/assets/87784067/12db179a-0019-4cb1-abd3-30fe3b79e19e)

**Comment by waxmask** [comment]
![image](https://github.com/AvaloniaUI/Avalonia/assets/87784067/7902267c-597a-469e-85b9-f02b3686cad6)
the problem only appears if you set the HorizontalAlignment to center value for this element. Take a look at the screenshot

**Comment by timunie** [comment]
Can't reproduce with the snippets provided. Please file a very minimal sample to reproduce.
